### DISCUSSION TITLE
Unable to change dashboard logo

### DISCUSSION THREAD

**Discussion by looped-monk** [saleor/saleor#10579] [General] [answered]
I've changed brand logos to my brand logos just to test if its working. I placed the logo svg's(logo-dark, logo-white etc) in dashboard/assets/images. However they aren't reflecting on the Dashboard end after running docker-compose up. The specific logo I wish to change is SideBar-logo-143 which is in Menuitem-root which I cant seem to find. Which files do we have to look at for branding for Saleor, Dashboard and Storefront ?

Strangely, the changes I made on api end for logo on saleor/templates reflect properly.

Update : 

I located AppLayout.TSX and placed the logo link inside Href logo. However , I get the following error.

**Cannot GET /http://mydomain:9000/dashboard/assets/images/logo-only.svg**

The Saleor logo on top left above menu bar which I wish to change still shows up.

**Comment by peelar** [comment, marked as answer]
Hello @narayanan-ka 👋 

I am afraid there is no way to change the logo in the Dashboard sidebar. The code for it is actually not even in the Dashboard itself, it's coming from the Saleor's UI Kit - [`macaw-ui`](https://github.com/saleor/macaw-ui) (see: `src/components/AppLayout/AppLayout.tsx`, line 180).

We've already had inquiries about that. I will let you know if that ever changes.

**Reply by looped-monk** [reply]
@peelar  Thank you for your response. It would be great if that happens because its happening on the homepage of api which is nice.

Does the Storefront also behave the same or I can change the logos there , if yes, where to look ?

**Reply by peelar** [reply]
You can't configure Storefront to change the logo, but if you are willing to modify the code, the logo is placed in `apps/storefront/components/Navbar/Navbar.tsx`.

**Reply by looped-monk** [reply]
Modifying the code - is that a good practice since Saleor has an App and plugin based system whereby changes can be made without bricking the code ?. In the end, since the storefront will be exposed to the end user, it should be easy to configure certain brand elements like logo, font style etc ? - just a thought

**Reply by Ghadje** [reply]
Hi @narayanan-ka

Actually, you can change the logo by changing the SVG file on the Salero's UI Kit - macaw-ui

**Reply by looped-monk** [reply]
Hi @Ghadje Thanks for the update. Will try. :-)

**Reply by looped-monk** [reply]
@Ghadje Connecting after a while. Could you help ?. I have placed my logos in /storybook/public and replaced the default logos (favicon, msite etc) with exactly the same dimensions and uploaded to my github npm packages for version 1.4.2. Only the favicon changes when i open the saleor dashboard. The default saleor logo doesn't get replaced. Anything that I'm missing here
